embedded/index.mjs

```
@@ -90,6 +90,37 @@ export class Embedded {
     }
   }
 
+  async setRegion(region) {
+    const regionUrl = this.getDomain(region);
+    try {
+      this.User.setUrl(regionUrl);
+      this.App.setUrl(regionUrl);
+      this.Integration.setUrl(regionUrl);
+      this.Tokens.setUrl(regionUrl);
+      this.Workflows.setUrl(regionUrl);
+      this.Events.setUrl(regionUrl);
+      this.Compliance.setUrl(regionUrl);
+      this.Logs.setUrl(regionUrl);
+      this.Credentials.setUrl(regionUrl);
+      this.Link.setUrl(regionUrl);
+      this.Analytics.setUrl(regionUrl);
+      this.HeadlessInstallation.setUrl(regionUrl);
+      this.url = regionUrl;
+    } catch (error) {}
+  }
+
+  getDomain(region) {
+    const DEFAULT_REGION = 'us';
+    const DOMAIN_MAP = {
+      us: baseUrl,
+      eu: euBaseUrl,
+    };
+
+    const key = region ? region.toLowerCase() : DEFAULT_REGION;
+
+    return DOMAIN_MAP[key];
```

Should we throw a helpful exception if an invalid region is passed in?